README.md

```diff
@@ -34,9 +36,47 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Quick Installation via Helm
+## Installation via Helm
+
+### Quick Installation
 
 StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+Then run the helm quick install script or proceed to section [Manual Helm Installation](#manual-helm-installation) for configuration options.
+```sh
+curl --fail https://raw.githubusercontent.com/stackrox/stackrox/master/scripts/quick-helm-install.sh | bash
+```
+You may use the `--small` option in order to install StackRox on smaller clusters with limited resources.
```

Comment:
Would be nice if you could err on the side of verbosity and add the complete command that the user can copy and paste.

```sh
curl ... | bash --small
```

I'm afraid `--small` wouldn't just work like that that's why I'm asking for you to solve this problem for users.

Reply:
Okay, I'll add another line the user can just copy and paste.

Reply:
For inspiration, here is the command suggested for installing brew. Maybe we could us the same form and curl parameters?

```sh
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

src: https://brew.sh/

Reply:
Well my current solution is to do
`curl --fail https://raw.githubusercontent.com/stackrox/stackrox/master/scripts/quick-helm-install.sh | bash -s -- -small`
I could change it to the way brew does it, but then I would change it in both cases to be consistent.

Reply:
Yes please, change both - see my other comment to check why.